vars/updatecli.txt

```diff
@@ -32,8 +33,16 @@
         values: "",
       )
     }
-    </code></pre>
 
+    // Note: to get an access token from a Github App instead of a Github user, you'll have to use this as credentials:
+    withCredentials([usernamePassword(
+      credentialsId: 'github-app-updatecli-on-jenkins-infra',
+      usernameVariable: 'USERNAME_NOT_USED', // Setting this variable is mandatory, even if of no use
+      passwordVariable: 'UPDATECLI_GITHUB_TOKEN'
+    )]) {
+      // ...
+    }
+    </code></pre>
```

Comment:
Note sure what is the intent of this block? Since the default behavior is to use a github app, I don't udnerstand the comment?

Reply:
You might want to use the withCredential above with a Personal Access Token, while this withCredentials syntax is for using a Github App Installation Access Token (not the same parameters used)

Reply:
Then maybe inverting the comment then? Because it tells the opposite of what you are saying 😅

Reply:
> You might want to use the withCredential above
... above this one, which is for a Github App

More clear here?